### PR TITLE
patch fix mongodb srv, tls support

### LINKED ISSUE #3971: Using using pyMongo srv tracker with latest Rasa docker image
Using latest rasa/rasa:latest-full docker image and getting the following error. Shouldn't the dnspython module come standard in the docker image? Or is this due to something else?:
```rasa_1           | 2019-07-10 04:06:20 INFO     root  - Starting Rasa Core server on http://localhost:5200
rasa_1           | 2019-07-10 04:06:20.650678: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
rasa_1           | 2019-07-10 04:06:21.221940: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2200000000 Hz
rasa_1           | 2019-07-10 04:06:21.277310: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x556f569865c0 executing computations on platform Host. Devices:
rasa_1           | 2019-07-10 04:06:21.277484: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
rasa_1           | [2019-07-10 04:06:22 +0000] [1] [ERROR] Experienced exception while trying to serve
rasa_1           | Traceback (most recent call last):
rasa_1           |   File "/usr/local/lib/python3.6/site-packages/sanic/app.py", line 1096, in run
rasa_1           |     serve(**server_settings)
rasa_1           |   File "/usr/local/lib/python3.6/site-packages/sanic/server.py", line 742, in serve
rasa_1           |     trigger_events(before_start, loop)
rasa_1           |   File "/usr/local/lib/python3.6/site-packages/sanic/server.py", line 604, in trigger_events
rasa_1           |     loop.run_until_complete(result)
rasa_1           |   File "uvloop/loop.pyx", line 1451, in uvloop.loop.Loop.run_until_complete
rasa_1           |   File "/usr/local/lib/python3.6/site-packages/rasa/core/run.py", line 176, in load_agent_on_start
rasa_1           |     None, endpoints.tracker_store, _broker
rasa_1           |   File "/usr/local/lib/python3.6/site-packages/rasa/core/tracker_store.py", line 38, in find_tracker_store
rasa_1           |     domain=domain, host=store.url, event_broker=event_broker, **store.kwargs
rasa_1           |   File "/usr/local/lib/python3.6/site-packages/rasa/core/tracker_store.py", line 205, in __init__
rasa_1           |     connect=False,
rasa_1           |   File "/usr/local/lib/python3.6/site-packages/pymongo/mongo_client.py", line 524, in __init__
rasa_1           |     res = uri_parser.parse_uri(entity, port, warn=True)
rasa_1           |   File "/usr/local/lib/python3.6/site-packages/pymongo/uri_parser.py", line 318, in parse_uri
rasa_1           |     raise ConfigurationError('The "dnspython" module must be '
rasa_1           | pymongo.errors.ConfigurationError: The "dnspython" module must be installed to use mongodb+srv:// URIs
```
**Content of endpoints file (endpoints.yml)**
```
action_endpoint:
  url: "http://localhost:5200/webhooks/twilio/webhook"

  tracker_store:
    store_type: mongod
    url: mongodb+srv://(etc.etc.etc.)
    db: [REDACTED]
    username: [REDACTED]
    password: [REDACTED]
```

relevant part of Docker-compose.yml:
```
version: '3.0'
  
services:
  rasa:
    image: rasa/rasa:latest-full
    networks: ['rasa-network']
    ports:
    - "5005:5005"
    volumes:
    - ./:/app
            #- "./models:/app/models"
            #- "./data:/app/data"
      #- "./rasa:/usr/local/lib/python3.6/site-packages/rasa"
    command: run --enable-api --model models --p 5200 --connector twilio --credentials credentials.yml --endpoints endpoints-twilio.yml
```

### PR DESCRIPTION
**Pull request by ortsaCniveK**
**Proposed changes**:
- re-adds support for mongodb+srv:// URIs as well as TLS support, as per PR #4004 and #4403; was lost due to regression
    > Fixes #3971 and #2968

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] updated the changelog

**Note:** Had some issues with the changelog, so I tried to fix it as best I could, but let me know if I can use the up to date changelog from master @tmbo @wochinge

### CODE REVIEW

**Comment by wochinge** [comment]
Hi @ortsaCniveK ,
thanks for your work and sorry for the delay, I was on vacation. Would you mind updating the pr so that we can merge it asap?

**Comment by ortsaCniveK** [comment]
@wochinge not sure what you mean by updating it, @tmbo mentioned they wanted this in a patch release?  #4403 didn't make the cut -- did you need me to merge with the current master branch?

**Comment by wochinge** [comment]
@ortsaCniveK 
Sorry, I wasn't very clear
By "updating" I meant resolving the merge conflict. However, I also saw that you are trying to merge into `master`. Can you please do the PR against `1.3.x` as Tom suggested?

**Comment by ortsaCniveK** [comment]
@wochinge just changed the base; will be fixing the merge conflict in a sec

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,7 +7,11 @@ Rasa Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
-
+[Unreleased] - `1.3.x`
```
Comment (wochinge) [inline CHANGELOG.rst]:
```suggestion
[Unreleased 1.3.4] - `1.3.x`_```

Comment (wochinge) [reply CHANGELOG.rst]: probably the line length below has to be adapted, too

File: CHANGELOG.rst [hunk 2]
```diff
@@ -7,7 +7,11 @@ Rasa Change Log
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Fixed
+-----
+- Fixed regression of TLS, SRV dependencies for PyMongo in ``requirements.txt``
```
Comment (wochinge) [inline CHANGELOG.rst]:
```suggestion
- Fixed regression of TLS, SRV dependencies for PyMongo
```

Comment (wochinge) [inline CHANGELOG.rst]:
```suggestion
- re-added TLS, SRV dependencies for PyMongo after regression
```

**Comment by ortsaCniveK** [comment]
@wochinge just committed your suggestions, I think we should be all set now 😓

File: CHANGELOG.rst [hunk 3]
```diff
@@ -7,7 +7,11 @@ Rasa Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
-
+[Unreleased 1.3.4] - `1.3.x`_```
```
Comment (wochinge) [inline CHANGELOG.rst]:
```suggestion
[Unreleased 1.3.4] - `1.3.x`_
```

Comment (wochinge) [reply CHANGELOG.rst]: please make sure that the underlining has the correct length. Otherwise travis will fail 🙄

File: CHANGELOG.rst [hunk 4]
```diff
@@ -7,7 +7,11 @@ Rasa Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
-
+[Unreleased 1.3.4] - `1.3.x`_
```
Comment (wochinge) [inline CHANGELOG.rst]: my bad - travis keeps on complaining. Let's just remove that for now. 
```suggestion
[Unreleased 1.3.4]
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -7,7 +7,11 @@ Rasa Change Log
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (wochinge) [inline CHANGELOG.rst]:
```suggestion
^^^^^^^^^^^^^^^^^^
```

File: CHANGELOG.rst [hunk 6]
```diff
@@ -7,6 +7,11 @@ Rasa Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
+[Unreleased 1.3.5]
```
Comment (wochinge) [inline CHANGELOG.rst]: I am so sorry, but could you please update this one more time? We had a busy week and cut a couple of bug fixes releases in the mean time 🙈

Comment (ortsaCniveK) [reply CHANGELOG.rst]: Alright, took a little work, but this PR is at least slated for 1.3.7.  Let me know if there's anything else needed to merge this.

**Comment by wochinge** [comment]
@ortsaCniveK Finally merged 🎉 Sorry that this took so long!

**Pull request merged**